Automattic/wc-calypso-bridge
# Add disabled tasks accordion component for free trial

Fixes #931 

This PR adds a new task at the bottom of the tasklist to notify the user that some tasks are disabled for the trial plan.

* Copied over experimental collapsible list as a **temporary** solution -- we need to release a new version from the core. There is a [PR](https://github.com/woocommerce/woocommerce/pull/36806) waiting for a review. Once we release the change, we can remove [this file](https://github.com/Automattic/wc-calypso-bridge/pull/940/files#diff-57aaff5cbeae99e43559d13f8008e016a24b83bb1c583524a3a3f16bb621f770)
* Waiting for a response for this [comment](https://github.com/Automattic/wc-calypso-bridge/issues/931#issuecomment-1421432169)

### Test Instructions


1. Make sure to copy over the latest changes in [woocmmerce-calypso-bridge-helper.php](https://gist.github.com/moon0326/cac46c70a2cee81b61faef517fef7178) and the plugin is activated.
2. Change [this line](https://gist.github.com/moon0326/cac46c70a2cee81b61faef517fef7178#file-a-woocommerce-calypso-bridge-helper-php-L33) to false in your local env. This forces your env to have free trial.
3. Go to WooCommerce > Home
4. Confirm the accordion component.

![Screen Shot 2023-02-16 at 1 19 23 PM](https://user-images.githubusercontent.com/4723145/219489168-98774f0d-84cc-4cbf-9e1a-f579fd7ed035.jpg)
